roles/openshift_health_checker/library/docker_info.py

```diff
@@ -0,0 +1,50 @@
+# pylint: disable=missing-docstring
+"""
+Ansible module for determining information about the docker host.
+"""
+
+from ansible.module_utils.docker_common import *
```

Comment:
> It's bad form in Python to import *, and preferred to import explicitly the names we need.

Thanks, will fix this

> Which brings us to the next question, why does DockerInfo needs to inherit from DockerBaseClass?

It is what I saw docker-related core modules do: https://github.com/ansible/ansible-modules-core/blob/devel/cloud/docker/docker_image.py#L259